RockWeb/Themes/NewSpring/assets/Lava/UI/child-items-swiper.lava

```diff
@@ -1,27 +1,53 @@
 {% if dataset and dataset != empty and cciid and cciid != empty %}
+
     {% assign cciid = cciid | AsInteger %}
     {% assign Item = dataset | PersistedDataset | Where:'Id', cciid | First %}
-
-    {% assign now = 'Now' | Date:'yyyy-MM-ddThh:mm:ss.000000' %}
-    {% assign startDate = Item.StartDateTime | Date:'yyyy-MM-ddThh:mm:ss.000000' %}
-    {% assign endDate = Item.EndDateTime | Date:'yyyy-MM-ddThh:mm:ss.000000' %}
+    {% assign now = 'Now' | Date:'yyyyMMddHHmmss' %}
+    {% assign startDate = Item.PublishDateTime | Date:'yyyyMMddHHmmss' %}
+    {% assign endDate = Item.EndDateTime | Date:'yyyyMMddHHmmss' %}
 
     {% comment %}
         Set current item slug so we can determine what the active slide should be
     {% endcomment %}
     {% assign currentSlug = 'Global' | Page:'Path' | Split:'/' | Last %}
     {% assign children = Item.Children %}
 
-    {% comment %}
-        If we're between the beginning/ending dates of this collection, make the latest item in the swiper active
-    {% endcomment %}
-    {% if now >= startDate and now < endDate %}
-        {% assign initialslide = children | Size | Minus:1 %}
-    {% else %}
-        {% assign initialslide = children | Where:'Slug', currentSlug | First | Property:'Index' | Minus:1 %}
+    {% if showfuturecontent != 'true' %}
```

Comment:
Added this parameter because according to OPS, we want to show pre-published content for devotionals so as to not prohibit users from reading entire studies when they would like to. Because of this, the dates are merely used to control when they show up in the app home feed, if I understand correctly.